Gopkg.lock

```diff
@@ -100,10 +100,13 @@
   version = "v1.3.0"
 
 [[projects]]
-  branch = "master"
+  branch = "travis-1.9"
```

Comment:
this is gonna away after alexey's PR

Reply:
If he merges his PR first I'll rebase. ;)